docs_src/design/ucr/Performance-Test-Harness.md

```diff
@@ -0,0 +1,52 @@
+## Performance Test Harness
+
+## Submitters
+- Cherry Wang (IOTech)
+
+## Changelog
+<!--
+List the changes to the document, incl. state, date, and PR URL.
+
+State is one of: pending, approved, amended, deprecated
+Date is an ISO 8601 (YYYY-MM-DD) string.
+PR is the pull request that submitted the change, including information such as the diff, contributors, and reviewers.
+
+E.g.:
+- [approved](URL of PR) (2022-04-01)
+- [amended](URL of PR) (2022-05-01)
+-->
+
+## Market Segments
+- All
+
+## Motivation
+The EdgeX Performance Test aims to evaluate the efficiency of the EdgeX services in handling a large volume of data transmission from the south side to the north side in a manufacturing context. This use case is motivated by the need to understand how EdgeX performs under heavy data loads and to identify potential areas for optimization.
+
+By providing the Performance Test Harness, users can execute the Performance Tests with their own hardware and devices environment to evaluate the capability of the system.
+
+## Target Users
+- System Developer
+- System Integrator
+
+## Description
+The EdgeX Performance Test involves the south side generating a substantial amount of data and transmitting it through EdgeX services to the north side. This process aims to simulate real-world scenarios where data flow between manufacturing devices and backend systems is intensive. The observation of this data transmission allows for an in-depth analysis of EdgeX's resource utilization patterns.
+
+## Existing solutions
+N/A
+
+## Requirements
+- A device / app-service profile template shall be defined that allows user to specify the device and app-services to be involved.
+- A configuration shall be defined that allows user to specify the number of device and app-services, server address, and etc. to run. ex. 10 devices per profile and 10 app-services.
```

Comment:
```suggestion
- A configuration shall be defined that allows user to specify the number of device and app-services, server address, and etc. to run. e.g. 10 devices per profile and 10 app-services.
```
Ex. is the abbreviation for "exercise". 

This is also used in three other places below, lines 40, 42, and 43.

Reply:
Updated.